src/ussd_nurse.js

```diff
@@ -377,6 +376,89 @@ go.app = function() {
             });
         });
 
+        self.add('state_save_nursereg', function(name) {
+        
+            var registrant_update_info = {
+                "details": {
```

Comment:
This will destroy their addresses and anything else in the details object currently. You can't patch a `details` json field. You need to read it first and update the object.